### ISSUE TITLE
Flashing 'Under Construction' Image - Accessibility

### ISSUE THREAD

**Issue by whitingx** [web-matters/web-matters.github.io#3]
Not sure how accessible friendly the current `Under Construction` placeholder image is on the site.

Could potentially go against guidelines for users with photosensitive epilepsy - http://webaim.org/techniques/images/#seizures - and is also missing it's `alt=""` text.

**Comment by decadecity** [comment]
I think it's probably under the [flash threshold](https://www.w3.org/TR/WCAG20/#general-thresholddef) but I'd agree that it would benefit from being less frantic.

**Action:** I'll change the animation rate down to 3 frames per second.

We've got a [null alt text](https://github.com/web-matters/web-matters.github.io/blob/f5db04c1017800531606f5ee7ef82f0ee06bdaf8/index.html#L13) on that image, I'm happy that it can be [safely ignored by assistive technology](https://www.w3.org/TR/WCAG20-TECHS/H67.html) but open to a case being made to add alt text.
